docs_user/modules/proc_configuring-ldap-with-domain-specific-drivers.adoc

```diff
@@ -0,0 +1,99 @@
+
+This involves two main steps:
+
+. Create the Kubernetes secret that holds the domain-specific LDAP configuration files that the {identity_service} uses. Each file within the secret corresponds to an LDAP domain.
+. Patch the `OpenStackControlPlane` custom resource (CR) to enable domain-specific drivers for the {identity_service} and mount a secret that contains the LDAP configurations.
+
+
+.Procedure
+
+. To create the `keystone-domains` secret that stores the actual LDAP configuration files that {identity_service} uses, create a local file that includes your LDAP configuration, for example, `keystone.myldapdomain.conf`:
++
+The following example file includes the configuration for a single LDAP domain. If you have multiple LDAP domains, create a configuration file for each, for example, `keystone.DOMAIN_ONE.conf`, `keystone.DOMAIN_TWO.conf`.
++
+[source,ini]
+----
+[identity]
+driver = ldap
+[ldap]
+url = ldap://<ldap_server_host>:<ldap_server_port>
+user = <bind_dn_user>
+password = <bind_dn_password>
+suffix = <user_tree_dn>
+query_scope = sub
+# User configuration
+user_tree_dn = <user_tree_dn>
+user_objectclass = <user_object_class>
+user_id_attribute = <user_id_attribute>
+user_name_attribute = <user_name_attribute>
+user_mail_attribute = <user_mail_attribute>
+user_enabled_attribute = <user_enabled_attribute>
+user_enabled_default = true
+# Group configuration
+group_tree_dn = <group_tree_dn>
+group_objectclass = <group_object_class>
+group_id_attribute = <group_id_attribute>
+group_name_attribute = <group_name_attribute>
+group_member_attribute = <group_member_attribute>
+group_members_are_ids = true
+----
++
+* Replace the values, such as `<ldap_server_host>`, `<bind_dn_user>`, `<user_tree_dn>`, and so on, with your LDAP server details.
+
+. Create the secret from this file:
++
+----
+$ oc create secret generic keystone-domains \
+  -n <namespace> \
+  --from-file=<keystone.DOMAIN_NAME.conf>
+----
++
+* Replace `<namespace>` with the namespace where your {identity_service} is deployed.
+* Replace `<keystone.DOMAIN_NAME.conf>` with the name of your local configuration file. If applicable, include additional configuration files by using the `--from-file` option. After creating the secret, you can remove the local configuration file if it is no longer needed, or store it securely.
++
+[IMPORTANT]
+The name of the file that you provide to `--from-file`, for example `keystone.DOMAIN_NAME.conf`, is critical. The {identity_service} uses this filename to map incoming authentication requests for a domain to the correct LDAP configuration. Ensure that `DOMAIN_NAME` matches the name of the domain you are configuring in the {identity_service}.
+
+. Patch the `OpenStackControlPlane` CR:
++
+----
+$ oc patch openstackcontrolplane <cr_name> \
+  -n <namespace> \
```

Comment:
We stopped adding namespace to oc commands. We ensure that project namespace is set at the beginning of procedure in **Adoption prerequisites**
https://openstack-k8s-operators.github.io/data-plane-adoption/#adoption-prerequisites_planning